src/olympia/reviewers/migrations/0027_backfill_needshumanreview.py

```diff
@@ -5,13 +5,7 @@
 
 
 def migrate_needs_human_review_field(apps, schema_editor):
-    NeedsHumanReview = apps.get_model('reviewers', 'NeedsHumanReview')
-    Version = apps.get_model('versions', 'Version')
-
-    for version in Version.unfiltered.filter(needs_human_review=True).filter(
-        Q(needshumanreview__isnull=True) | Q(needshumanreview__is_active=False)
-    ):
-        NeedsHumanReview.objects.create(version=version)
+    pass
```

Comment:
I was getting migration conflicts because of this old migration, given it's a data migration and has already been executed in every env it shouldn't matter if it's blank.